packages/shorebird_cli/test/src/commands/preview_command_test.dart

```diff
@@ -54,6 +56,101 @@ void main() {
     late ShorebirdValidator shorebirdValidator;
     late PreviewCommand command;
 
+    String runnerPath() {
+      final platformName = ReleasePlatform.ios.name;
+      return p.join(
+        previewDirectory.path,
+        '${platformName}_${releaseVersion}_$iosArtifactId.app',
+      );
+    }
+
+    String aabPath() {
+      final platformName = ReleasePlatform.android.name;
+      return p.join(
+        previewDirectory.path,
+        '${platformName}_${releaseVersion}_$androidArtifactId.aab',
+      );
+    }
+
+    String apksPath() {
+      final platformName = ReleasePlatform.android.name;
+      return p.join(
+        previewDirectory.path,
+        '${platformName}_${releaseVersion}_$androidArtifactId.apks',
+      );
+    }
+
+    File setupIOSShorebirdYaml() => File(
+          p.join(
```

Comment:
Naming consistency nit: `setupIOSShorebirdYaml` vs `createAndroidShorebirdYaml`. These should line up.